analysis/src/python/data_collection/stepik/README.md

```diff
@@ -0,0 +1,37 @@
+## Stepik client
+
+This module contains Stepik client ``StepikClient`` implementation.
+
+### Supported requests
+
+Implementation was based on open [API](https://stepik.org/api/docs/) provided by Stepik.
```

Comment:
It would be great if you add some description in each readme what we support/ I mean what information a user can gather by our tool